payment_paybox/README.md

```diff
@@ -0,0 +1,21 @@
+## Paybox payment acquirer module for Odoo ##

```

Comment:
Can you please make the README in RST format (the only one supported by Odoo), and using this template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst?